### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by spencerldixon**
Implements a new design of the unep-wcmc website, probably breaks a lot of things too...

### CODE REVIEW

File: app/views/shared/_hero.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<header class="hero background--fishing">
+  <div class="hero__card">
+    <div class="hero__title">
+      <h1>Knowledge that makes a world of difference</h1>
+      <br>
+      <p>UNEP-WCMC is a world leader in biodiversity knowledge.  It works with scientists and policy makers worldwide to place biodiversity at the heart of environment and development decision-making to enable enlightened choices for people and the planet.  </p>
```
Comment (lucidstack) [inline app/views/shared/_hero.html.erb]: Is this staying as UNEP-WCMC or should we change it?

File: app/views/shared/_latest_news.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<section class="row">
+  <div class="container cards-row">
+    <% Comfy::Cms::Site.find_by_identifier('news').pages.root.children.published.includes(blocks: [:files]).reverse.take(6).each do |page| %>
+      <div class="card card--third card--news">
+        <% #if page_file(:news_img, page).present? %>
```
Comment (lucidstack) [inline app/views/shared/_latest_news.html.erb]: Comments!

File: app/assets/stylesheets/components/biography.scss [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  }
+}
+
+.biography__photo {
+  min-width: 200px;
+  width: 30%;
+  margin: auto;
+  @media (max-width: $tablet){
+    min-width: 200px;
+    display: -webkit-box;
```
Comment (spencerldixon) [inline app/assets/stylesheets/components/biography.scss]: Can't use the flexbox mixin within media queries, that's why this bit is different :(

File: app/assets/stylesheets/components/logo.scss [hunk 4]
```diff
@@ -0,0 +1,36 @@
+@import 'utilities/colors';
+@import 'utilities/breakpoints';
+@import 'utilities/flexbox';
+
+$logo_breakpoint: $tablet;
+
+.logo {
+  @extend %flexbox;
```
Comment (spencerldixon) [inline app/assets/stylesheets/components/logo.scss]: Because logo is the block that contains the logo and the text arranged in sexy ways

Comment (lucidstack) [inline app/assets/stylesheets/components/logo.scss]: Alrite!

**Pull request merged**